open-science-projects.md

```diff
@@ -224,7 +224,9 @@ Students also learn about the process of writing a collaborative paper by attend
 ## Mistakes and benefits
 
 We have made plenty of mistakes.
-Students have spent weeks stuck on problems that other team members know how to solve, have written large chunks of code that aren't possible to incorporate into the larger project, and have cobbled together their own (undocumented) data structures.
+Students have spent weeks stuck on problems that other team members know how to solve, have written large chunks of code that aren't possible to incorporate into the larger project, and have cobbled together their own (undocumented and barely usable) data structures.
+Offering related projects over subsequent years helped us to learn from our previous mistakes, and to make new ones instead.
```

Comment:
Maybe would reword this to "helped us to learn from our previous mistakes, and PROGRESS TO new ones instead." - it gives a better sense of forward progress than an immovable brick wall of mistakes preventing progression?

Reply:
Yes please!

Reply:
edited.